packages/Layout/footer/src/FooterCore.tsx

```diff
@@ -15,9 +15,8 @@ interface FooterCoreComponentProps {
   alt?: string;
   className?: string;
 }
-
 const defaultProps: Partial<FooterCoreComponentProps> = {
-  children: '@ 2018 AXA Tous droits réservés',
+  children: `© ${new Date().getFullYear()} AXA Tous droits réservés`,
```

Comment:
You will need to inject children on stories in order to make storyshot working the next year, i think

Reply:
Stories already inject children ;)
There's no snapshot change in with this PR.

Reply:
I like your PR, but I think every projects every year will have tests broken on the next year.

I do not find a solution.

Do you have an idea ?

Reply:
It's a default prop, and update broken tests because we change year is not really a big issue for me.

And so, put 2018 by default is worst for me.

Reply:
Or you just remove the year if you prefer, but in this case you will remove the year for many projects.

Reply:
Or we just put 2020 instead of 2018 by default.

Reply:
If you change the year manually every year, you might as well keep the script...

Reply:
@samuel-gomez any idea or opinion ?

Reply:
But it's default props, I think we should use the script version and explain it in the changelog.

Reply:
By default, it's better to use the script version. If the user wants to override it, he can